dandi/dandiapi.py

```diff
@@ -374,7 +402,11 @@ class Config:
 
 
 class Version(APIBase):
-    """The version information for a Dandiset retrieved from the API"""
+    """
+    The version information for a Dandiset retrieved from the API.
+
+    This class should not be instantiated directly.
```

Comment:
please extend with an example on how should it be instantiated

Reply:
End-users should only retrieve instances from the properties & methods that return `Version` objects; do you want me to list every single one?

Reply:
No, but at least one with a description of which class instances have methods to get instances